images/installer-libvirt/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# This Dockerfile is a used by CI to publish openshift/origin-v4.0:installer
```

Comment:
Comment might need an update

Reply:
Good catch, updated the pullspec and the purpose